core/server/master/src/main/java/alluxio/master/metastore/caching/CachingInodeStore.java

```diff
@@ -445,17 +452,18 @@ protected void flushEntries(List<Entry> entries) {
           continue;
         }
         try (LockResource lr = lockOpt.get()) {
-          if (entry.mValue == null) {
+          Long value = entry.mValue;
```

Comment:
Is this just to clean up code style?

Reply:
It was actually to prevent a race condition in `InodeStoreTest` due to a violated locking assumption (calling removeChild() without holding a lock on the removed edge)

I've updated the test to acquire the necessary locks, and I've updated the InodeStore javadoc to remind the user of the locking requirement